test/e2e/rich-data-table.test.js

```diff
@@ -303,6 +303,27 @@ describe('vl-rich-data-table', async () => {
         await assert.eventually.isTrue(filter.isDisplayed());
     });
 
+    it('Als gebruiker met een klein scherm, kan ik de filter openen als modal, gebruiken en terug sluiten', async () => {
+        const rect = await driver.manage().window().getRect();
+        const originalWidth = rect.width;
+        await driver.manage().window().setRect({height: rect.height, width: 750}); // small size
```

Comment:
geen mogelijkheid om ergens op te vragen wat "small" is voor webuniversum? want als ze dat veranderen moet dit ook veranderen denk ik?

Reply:
Lijkt me wel mogelijk met een complexe zoektoch in de stylesheets. Maar op zich zal de test dan falen, wat me op zich wel voldoende lijkt, niet?

Reply:
Ik heb even gezocht of het niet mogelijk is om te zeggen tegen de browser dat hij mobile moet simuleren, net zoals bij de developer tools, maar niet meteen iets gevonden.